lib/participant_profile_deduplicator.rb

```diff
@@ -101,6 +102,15 @@ def reconcile_schedules!
     update_primary_profile_schedule(profile_with_correct_schedule.latest_induction_record.schedule)
   end
 
+  def update_mentored_ects!
+    return unless [primary_profile, duplicate_profile].all?(&:mentor?)
+
+    duplicate_profile.mentee_profiles.each do |mentee|
+      mentee.update!(mentor_profile_id: primary_profile.id)
+      mentee.induction_records.each { |induction_record| induction_record.update!(mentor_profile_id: primary_profile.id) }
```

Comment:
bah sorry I realised something I missed again, I think we need to only update the IRs `mentor_profile_id` **IF** the `mentor_profile_id` equals the one we want to delete. As we might be accidentally be removing history here:  
- Someone has a `nil` `mentor_profile_id` in the beginning
- Someone has a different mentor profile and transferred to the current one later.

So maybe safest way is to just find the IRs where the `mentor_profile_id` equal the one changing and updating those only, and ensuring that the profile has the old one as well before updating, sorry again! Hopefully this catches all variations